tempodb/encoding/vparquet3/block_traceql.go

```diff
@@ -448,10 +448,10 @@ const (
 	columnPathServiceStatsSpanCount    = "ServiceStats.key_value.value.SpanCount"
 	columnPathServiceStatsErrorCount   = "ServiceStats.key_value.value.ErrorCount"
 	columnPathResourceAttrKey          = "rs.list.element.Resource.Attrs.list.element.Key"
-	columnPathResourceAttrString       = "rs.list.element.Resource.Attrs.list.element.Value"
-	columnPathResourceAttrInt          = "rs.list.element.Resource.Attrs.list.element.ValueInt"
-	columnPathResourceAttrDouble       = "rs.list.element.Resource.Attrs.list.element.ValueDouble"
-	columnPathResourceAttrBool         = "rs.list.element.Resource.Attrs.list.element.ValueBool"
+	columnPathResourceAttrString       = "rs.list.element.Resource.Attrs.list.element.Value.list.element"
```

Comment:
this shouldn't increase our nesting levels, correct?

Reply:
Yes. The corresponding schema looks like this:
```
REQUIRED group Value (LIST) {
    REPEATED group list {
        REQUIRED BYTE_ARRAY element (STRING);
    }
}
```
Since both `Value` and `element` are required this does change the max definition level of the column